src/main/java/frc/robot/commands/ShooterPID.java

```diff
@@ -0,0 +1,28 @@
+package frc.robot.commands;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.ShooterSystem;
+
+public class ShooterPID extends Command {
+    private ShooterSystem shooterSystem;
+    private final double targetRPM;
+    public ShooterPID(ShooterSystem shooterSystem, double targetRPM){
+        this.shooterSystem = shooterSystem;
+        this.targetRPM = targetRPM;
+    }
+    public void initialize() {
+        shooterSystem.rotatePID();
```

Comment:
```suggestion
        shooterSystem.rotatePID(targetRPM);
```

we need to use that in our command and give the method this value

Reply:
done